main.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'app'
+
+def options
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def prompt
+  puts 'Welcome to the University School Library App!'
+  loop do
+    options
+    option = gets.chomp.to_i
+    break if option == 7
+
+    call_option(option)
+  end
+end
+
+def call_option(option)
+  case option
+  when 1
+    list_books
+  when 2
+    list_people
+  when 3
```

Comment:
- [ ] Great job working on the project thus far! However, when I try to create a `student` or `teacher`, I get this error and cannot successfully create a person or test a rental.
<img width="1124" alt="Screenshot 2023-12-08 at 20 47 57" src="https://github.com/Obimbo07/school-library/assets/64774936/10bb6537-d1b8-454b-ba8d-ed9dc3d3c4fe">